ligo/helpers.mligo

```diff
@@ -37,9 +37,19 @@
    *)
 let sqrt_price_move_x (liquidity : nat) (sqrt_price_old : x80n) (dx : nat) : x80n =
     (* floordiv because we want to overstate how much this trade lowers the price *)
-    {x80 = floordiv
-        (Bitwise.shift_left (liquidity * sqrt_price_old.x80) 80n)
-        ((Bitwise.shift_left liquidity 80n) + dx * sqrt_price_old.x80)}
+    let sqrt_price_new =
+        {x80 = floordiv
+            (Bitwise.shift_left (liquidity * sqrt_price_old.x80) 80n)
+            ((Bitwise.shift_left liquidity 80n) + dx * sqrt_price_old.x80)
+        } in
+#if DEBUG
+    let _ : unit =
+        if sqrt_price_new <= sqrt_price_old
```

Comment:
@dcastro  Is there a case where the new price stay the same?

Reply:
Yeah, technically you _can_ swap 0 tokens (tho I'm not sure why anyone would do this lol)